### PR TITLE
[stable/kong] use custom server block for health-checking

### PR DESCRIPTION
**Pull request by hbagdi**
Signed-off-by: Harry Bagdi <harrybagdi@gmail.com>

#### What this PR does / why we need it:

Uses custom server block for health-checking.
Meaning, if RBAC is turned on, then , the health-checks will not fail.

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by hbagdi** [comment]
ping @shashiranjan84

File: stable/kong/values.yaml [hunk 1]
```diff
@@ -305,8 +305,8 @@ readinessProbe:
 livenessProbe:
   httpGet:
     path: "/status"
-    port: admin
-    scheme: HTTPS
+    port: metrics
```
Comment (shashiranjan84) [inline stable/kong/values.yaml:308]: Are we registering `metrics` port somewhere?

Comment (hbagdi) [reply stable/kong/values.yaml:308]: Yes.
https://github.com/helm/charts/blob/ce04b0bba646957c632435adc5a2212834055fdd/stable/kong/templates/deployment.yaml#L193

Comment (shashiranjan84) [reply stable/kong/values.yaml:308]: @hbagdi test failing otherwise LGTM

**Comment by hbagdi** [comment]
@shashiranjan84 Should be good now.

**Review by shashiranjan84 — APPROVED** [review]
/lgtm

**Pull request merged**